pyscriptjs/tests/integration/support.py

```diff
@@ -299,9 +343,17 @@ class ConsoleMessageCollection:
       console.error.*
       console.warning.*
 
+      console.js_error.*    this is a special category which does not exist in the
+                            browser: it prints uncaught JS exceptions
+
       console.all.*         same as above, but considering all messages, no filters
```

Comment:
[suggestion] Maybe we should reword the doc string for console.all? Since the above is now different.

Reply:
thanks, fixed by 9fb4ff5